frontend/src/components/common/InnerTable.tsx

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core';
+import SimpleTable, { SimpleTableProps } from './SimpleTable';
+
+const useStyles = makeStyles({
+  root: {
+    border: 'none',
+    '& .MuiTableCell-head': {
+      background: 'none',
+    },
+  },
+});
+
+export default function InnerTable(props: SimpleTableProps) {
```

Comment:
I'm not sure this needs to be a component if it's used only in one place. Probably better to move the two style rules there where it's used instead.

Reply:
I answered in the other comment about its use. I will remove the commented out code, though.